### DISCUSSION TITLE
Default binding for __fzf_search_current_dir conflicts with default fish binding for autosuggestions

### DISCUSSION THREAD

**Discussion by benjamineskola** [PatrickF1/fzf.fish#99] [Archive] [answered]
By default `__fzf_search_current_dir` is bound to `Control+F`, but this is already the binding for [accepting an autosuggestion](https://fishshell.com/docs/current/index.html#autosuggestions).

Perhaps this binding could be moved to `Control+Alt+F`, in common with the bindings for searching the git log and git status?

(I realise that it's possible to fix this individually by setting the custom bindings, but it seems to me that it would be preferable not to conflict by default.)

**Comment by PatrickF1** [comment]
Oh dear, I did not realize Ctrl+F was used for such an important feature of fish. Let me sit on this and consider alternatives....
It's hard because whatever I do will greatly impact all the users. Thank you for bringing this up @benjamineskola!

**Comment by PatrickF1** [comment]
Hi again. I've sat on this and I'm going to keep the current key bindings. Reasoning
1) The right arrow can also be used to accept an autosuggestion, not just Ctrl+f
2) I highly doubt most of fzf.fish users know, much less use, Ctrl+F to accept an autosuggestion
3) It's too disruptive to force people to change
4) Ctrl+Alt+F is a bit annoying to hit for people with smaller hands and there's not really a better mnemonic key 

Thank you for the feedback though. Keep it coming! :)

**Comment by dessalines** [comment]
I ran into this too, the fish default auto-complete is Control-F, and its muscle memory for me now. This would probably do better to use `control-O` rather than overriding a default fish shell standard.

**Comment by PatrickF1** [comment]
Hi @dessalines, thanks for your feedback. I'm sorry my key bindings conflict with your muscle memory. Unfortunately, changing it at this point is too late because the change would be backwards incompatible with all current users. May I suggest customizing the key bindings?

**Comment by dessalines** [comment]
Yes I will do that, but I still think it would be better for a plugin to not override a fish shell default.

**Comment by PatrickF1** [comment, marked as answer]
@benjamineskola @dessalines The default binding is now Ctrl+Alt+F thanks to your feedback! See https://github.com/PatrickF1/fzf.fish/releases/tag/v7.0.
